### PR TITLE
Update Traceroute README

### PR DESCRIPTION
**Pull request by nflath**
Start of updating the Traceroute PacketIn readme in order to test VRF behaviour.

### CODE REVIEW

File: feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md [hunk 1]
```diff
@@ -1,34 +1,124 @@
...
+At the start of each of the following scenarios, ensure:
+
+*   All ports are up and baseline is reset as above.
+
+Unless otherwise specified, all the tests below should use traffic with
+`dscp_encap_a_1`.
+
+### Test-1
+
+Tests that traceroute for a packet coming from the cluster respects
```
Comment (xw-g) [inline feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md:40]: "a packet coming from the cluster respects" -> "a packet with TTL=1 matched the vrf selection policy". Basically we want to have a summary if the packet get matched to a vrf selection policy or matched by the P4RT rules.

File: feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md [hunk 2]
```diff
@@ -1,34 +1,124 @@
...
+### Test-8
 
+Tests that traceroute for a packet with no route reports a miss.
 
-## Config Parameter coverage
+### Test-9
 
-*    /components/component/integrated-circuit/config/node-id
-*    /interfaces/interface/config/id
+Regionalization cases (TBD)
```
Comment (xw-g) [inline feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md:87]: redact it.

File: feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md [hunk 3]
```diff
@@ -1,34 +1,124 @@
...
+*   Send packets to DUT port-1 with TTL=1. The outer v4 header has the destination
+    addresses 138.0.11.8. verify that packets with TTL=1 are received by the client.
+
+*   Verify that the packets have both ingress_singleton_port and egress_singleton_port metadata set.
+egress_singletone_port should have port 2 with weight 1, port 3 weight 3, and port 4 with weight 6.
+
+
+### Test-2
+
+Tests that traceroute for a packet coming from the cluster that should
```
Comment (xw-g) [inline feature/experimental/p4rt/otg_tests/traceroute_packetin_test/README.md:52]: describe should the packet get matched to a vrf selection policy or matched by the P4RT rules.

**Comment by nflath** [comment]
Updaetd based on cmoments

**Pull request closed without merging**